addons/sale_loyalty/views/sale_order_views.xml

```diff
@@ -7,6 +7,17 @@
         <field name="inherit_id" ref="sale.view_order_form"/>
         <field name="priority">10</field>
         <field name="arch" type="xml">
+            <div name="button_box" position="inside">
+                <button
+                    class="oe_stat_button"
+                    type="object"
+                    name="action_view_gift_cards"
```

Comment:
Name should be first attribute if there is no string :see_no_evil: 
see [Guildlines](https://www.odoo.com/odoo/knowledge/8790) for rationale and positioning of the attributes